README.md

```diff
@@ -21,7 +21,7 @@ For more information see the [GWB site](https://geodynamicworldbuilder.github.io
 
 ## Frequently Asked Questions (FAQ)
 ### What are the minimum requirements to compile the library and tools?
-For GWB version 0.6.0 only a C++14 compiler and CMake version higher than 2.8.12 are required to compile the library tools. Note that for some optional features (such as certain wrappers and unity build) a higher version of CMake and other dependencies may be needed (e.g., a Fortran compiler to compile the Fortran wrapper).
+For GWB version 1.0.0 only a C++14 compiler and CMake version higher than 2.8.12 are required to compile the library tools. Note that for some optional features (such as certain wrappers and unity build) a higher version of CMake and other dependencies may be needed (e.g., a Fortran compiler to compile the Fortran wrapper).
```

Comment:
Make your life easier by mentioning the specific number in as few places as possible:
```suggestion
Currently, GWB requires only a C++14 compiler and CMake version higher than 2.8.12 to compile the library tools. Note that for some optional features (such as certain wrappers and unity build) a higher version of CMake and other dependencies may be needed (e.g., a Fortran compiler to compile the Fortran wrapper).
```